usage/app/model/MediaUsage.scala

```diff
@@ -96,4 +96,21 @@ object MediaUsage {
     None,
     printUsage.dateAdded
   )
+
+  def build(atomWrapper: AtomWrapper, contentWrapper: ContentWrapper): MediaUsage = {
```

Comment:
Does it make sense to modify `ContentWrapper` so that it can contain both "content" and "atoms"?